health/micro-ui/web/micro-ui-internals/packages/css/src/pages/employee/usertracking.scss

```
@@ -0,0 +1,51 @@
+.user-tracking-inbox-table-wrapper {
+  .user-tracking-inbox-table {
+    .rdt_TableBody {
+      border: none !important;
+      border-bottom: 0.063rem solid var(--digitv2-lightTheme-divider) !important;
+    }
+    overflow-x: auto !important;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)